velox/docs/develop/types.rst

```diff
@@ -2,61 +2,75 @@
 Types
 =====
 
-Velox supports scalar types and complex types. These types cover most of the
-Presto and Spark data types.
+Velox supports a fixed set of physical types, an extensible set of logical types,
```

Comment:
Let's not mix in complex typed and Presto types into physical and logical types. Let's clarify that physical types determine in-memory layout of the data while logical types add semantics.